src/components/ButtonsPanel.js

```diff
@@ -0,0 +1,38 @@
+import Button from './Button';
+
+const ButtonsPanel = () => (
+  <div>
+    <div>
```

Comment:
**[OPTIONAL]** 
* [ ] Why not store the button names in an array and loop over the array and render Button. Then each time you are rendering a button, you pass the button name as the prop to the button. 
`buttonNames.map(name=> <button name={name} />)`
All you go to do it to get the order of the name right. This gives a much cleaner code.